tdi_python/tdicli.py

```diff
@@ -894,26 +894,8 @@ def add_from_json(self, entry_blob):
             print("Input must be string produced by dump command for this table.")
 
 
-    def _create_attributes(self, key_fields, data_fields={}):
-        method_name = "dynamic_key_mask_set"
-        if method_name not in self._c_tbl.supported_commands:
-            return
-        param_str, param_docstring, parse_key_call, parse_data_call, param_list = self._make_core_method_strs(method_name, key_fields, data_fields)
-
-        code = '''
-def {}(self, {} ):
-    """Add dynamic mask attribute to {} .
-
-    Parameters:
-    {}
-    """
-    parsed_keys, parsed_data = self._c_tbl.parse_str_input("{}", {}, {})
-    if parsed_keys == -1:
+    def _create_attributes(self, key_fields={}, data_fields={}):
```

Comment:
Yes. That's why removed everything from inside from this. Currently this takes in key and data fields of the table. Eventually this will take in "attribute fields". Do you have any other way to override this?

Reply:
Can you add comments on _create_attributes function? since currently we don't use this function. What kind use case is this function called or how to use this function?
Do you have example on this function on target?

Reply:
Got it. thanks